cardstock/uiButton.py

```diff
@@ -228,7 +233,9 @@ def SetProperty(self, key, value, notify=True):
     def UpdatePropKeys(self, style):
         # Custom property order and mask for the inspector
         if style in ("Border", "Borderless"):
-            self.propertyKeys = ["name", "title", "style", "rotation", "position", "size"]
+            self.propertyKeys = ["name", "title", "style", "rotation", "position", "size", "text_color"]
```

Comment:
I've been keeping rotation, position, size as the last items.
```suggestion
            self.propertyKeys = ["name", "title", "style", "text_color", "rotation", "position", "size"]
```